locust/static/locust.js

```diff
@@ -220,8 +228,13 @@ function initChart(stat) {
 }
 
 function updateCharts(stats) {
-
+    var rsp_chart, rps_data, now, date, avt_chart, avt_data;
```

Comment:
Oops, typo. I just added those in really quick when I realized we were creating variables in the global scope. Fixed.